java/src/org/openqa/selenium/remote/CapabilityType.java

```diff
@@ -47,6 +47,10 @@
   String HAS_TOUCHSCREEN = "hasTouchScreen";
   String OVERLAPPING_CHECK_DISABLED = "overlappingCheckDisabled";
   String STRICT_FILE_INTERACTABILITY = "strictFileInteractability";
+  String TIMEOUTS = "timeouts";
+  String IMPLICIT_TIMEOUT = "implicit";
```

Comment:
These are really meant to be the keys in the capabilities, not the keys of values within the capabilities